hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2042,7 +2042,8 @@
               anymore.
             '''
       milestone: V2
-      tests: []
+      tests: ["chip_sw_sram_ctrl_scrambled_access",
```

Comment:
> Drive the CPU to perform random accesses to both RAMs and verify these operations
              complete successfully by using the backdoor interface

The test desc mentions that we will verify the access via backdoor interface. I didn't find that in the C test.
We want to test the integration of OTP and sram. we need to prove sram uses the nonce and key from OTP.
In order to prove that, we can backdoor write a value with key and nonce to the sram mem on SV side and read that expected value from C side.

I can help creating the backdoor write task. Not sure if you know how to communicate between C and SV

Reply:
Thanks @weicaiyang, help with the backdoor write task would be very useful.

Reply:
@davebwil done, I added the support in #10613